### PR TITLE
Update doc: SB3-Contrib

### PR DESCRIPTION
**Pull request by araffin**
## Description

Update doc to mention sb3-contrib and update internal variable names to match the one discussed in https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/pull/13.
I also fixed a minor bug when saving DQN q-network alone.

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (update in the documentation)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [x] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: tests/test_save_load.py [hunk 1]
```diff
@@ -408,6 +408,82 @@ def test_save_load_policy(tmp_path, model_class, policy_str):
         os.remove(tmp_path / "actor.pkl")
 
 
+@pytest.mark.parametrize("model_class", [DQN])
```
Comment (AdamGleave) [inline tests/test_save_load.py:411]: Why do we have this as a parameter when we only set it to one value? Upgrade potential?

**Review by AdamGleave — COMMENTED** [review]
Few wording changes and one question about test parameter, otherwise LGTM

File: tests/test_save_load.py (hunk 1 above)
Comment (araffin) [reply tests/test_save_load.py:411]: hmm, I wrote that test when I found the bug reviewing QR-DQN PR on the contrib repo.
So, I think my idea was that QR-DQN will be part of SB3 main repo at some point, hence the parameter ;).

**Comment by araffin** [comment]
> Few wording changes and one question about test parameter, otherwise LGTM

thanks for the suggestions =) (I will update SB3 contrib readme too afterward)

**Review by AdamGleave — APPROVED** [review]
LGTM

**Pull request merged**